packages/sol-cov/test/collect_contracts_data_test.ts

```diff
@@ -3,16 +3,17 @@ import * as _ from 'lodash';
 import 'mocha';
 import * as path from 'path';
 
-import { collectContractsData } from '../src/collect_contract_data';
+import { ZeroExArtifactAdapter } from '../src/artifact_adapters/0x';
 
 const expect = chai.expect;
 
 describe('Collect contracts data', () => {
```

Comment:
Rename this so it's `ZeroExArtifactAdapter` specific. Also rename the test file.